### PR TITLE
Editorial updates to overview documents.

### PR DESCRIPTION
**Pull request by natke**

### CODE REVIEW

File: articles/TOC.yml [hunk 1]
```diff
@@ -6,6 +6,8 @@
     href: welcome.md
   - name: Five questions about quantum computing
     items:
+    - name: Overview
```
Comment (cjgronlund) [inline articles/TOC.yml]:
```suggestion
 
```

File: articles/TOC.yml [hunk 2]
```diff
@@ -6,6 +6,8 @@
...
+      href: overview/index.md
```
Comment (cjgronlund) [inline articles/TOC.yml]:
```suggestion

```

Comment (cjgronlund) [reply articles/TOC.yml]: PMs requested we remove this from TOC. Let's do it for now.

File: articles/overview/quantum-computers.md [hunk 3]
```diff
@@ -12,7 +12,9 @@ uid: microsoft.quantum.overview.computers
 
 What we can do with a quantum computer that can't be done with a classical one?
```
Comment (cjgronlund) [inline articles/overview/quantum-computers.md]:
```suggestion
What can we do with a quantum computer that can't be done with a classical one?
```

File: articles/TOC.yml (hunk 2 above)
Comment (natke) [reply articles/TOC.yml]: Sure

**Pull request merged**